sdk/monitor/azure-monitor-query/README.md

```diff
@@ -147,12 +147,14 @@ query = """AzureActivity | take 5"""
 
 try:
     response = client.query_resource(os.environ['LOGS_RESOURCE_ID'], query, timespan=timedelta(days=1))
-    if response.status == LogsQueryStatus.PARTIAL:
+    if isinstance(response, LogsQueryResult):
```

Comment:
Is it possible to use a [tagged union](https://mypy.readthedocs.io/en/stable/literal_types.html#tagged-unions) here so we don't have to do isinstance checks for typing?

Reply:
Thanks for the pointers. Updated the `status` values to user Literals with the corresponding enum values. Now, the `response.status` checks can narrow down the typing and we no longer need the `isinstance` checks.